src/jetstream/passthrough.go

```diff
@@ -525,6 +525,7 @@ func (p *portalProxy) doRequest(cnsiRequest *interfaces.CNSIRequest, done chan<-
 	} else if res.Body != nil {
 		cnsiRequest.StatusCode = res.StatusCode
 		cnsiRequest.Status = res.Status
+		cnsiRequest.ResponseHeader = res.Header
```

Comment:
Think there was a reason why jetstream never forwarded all response headers, which should do this in a targeted way with the specific headers that are needed instead

Reply:
Update - https://github.com/cloudfoundry/stratos/blob/f34f428ca77527e47072ae9b4ed47eec74f1deac/src/jetstream/plugins/userinfo/uaa_user.go#L171

When proxying we don't want things like connection or cookie being sent back to the browser but kept between server and destination.

Reply:
Ok, so just getting the `content-length` should work in our case ?

Reply:
I've removed the for loop and left the copy of `x-content-length` from response header.
This has also fixed the bug in downloading file reported [here](https://github.com/epinio/ui/pull/261#pullrequestreview-1545207239)